backend/app/token_whitelists/storage_helpers/storage_helpers.py

```diff
@@ -0,0 +1,17 @@
+from typing import Union
```

Comment:
We started using the `adapter` subfolder in other app-modules (`treasury`, `spread`), to list modules that take care of adapters. These adapter modules are named by the external service they are relating to.

Here, `storage_helpers.py` is really an adapter to Redis. I thus suggest: `token_whitelists/adapters/redis.py`. No need for a folder, btw.